v3/plugins/redhat/php/latest/meta.yaml

```diff
@@ -9,10 +9,10 @@ type: VS Code extension
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 repository: https://github.com/bmewburn/vscode-intelephense
 category: Language
-firstPublicationDate: "2019-04-16"
+firstPublicationDate: "2019-06-13"
 spec:
   containers:
     - image: "eclipse/che-remote-plugin-php7:next"
       memoryLimit: "1000Mi"
   extensions:
-    - https://github.com/che-incubator/vscode-intelephense/releases/download/v1.0.13/bmewburn.vscode-intelephense-client-1.0.13.vsix
+    - https://github.com/che-incubator/vscode-intelephense/releases/download/v1.0.14/bmewburn.vscode-intelephense-client-1.0.14.vsix
```

Comment:
No New line in the end 
@vrubezhny please take a look at formatting

Reply:
does that matter? is the yaml so brittle it breaks w/o a trailing newline?

Reply:
No, but `cat` does ;)